base/commandLine.cpp

```diff
@@ -752,6 +756,24 @@ static void listAudioDevices() {
 	}
 }
 
+/** Display all games in current directory */
+static void autoDetect(){
+	//Current directory
```

Comment:
@iskrich The code formatting guidelines for ScummVM are at http://wiki.scummvm.org/index.php/Code_Formatting_Conventions.